api/WEBGL_compressed_texture_pvrtc.json

```diff
@@ -6,10 +6,10 @@
         "spec_url": "https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_pvrtc/",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "28"
```

Comment:
The commit you linked says this might not actually be supported on desktop. It depends on the hardware I guess, and perhaps the OS.

Not sure how to represent this. Is there any pattern we can copy?

Reply:
I wasn't certain about the desktop support either, especially since the existing data claims that BCD supports it on desktops and it was set to `true` originally, which came from #3704.  However, the documentation talks about support almost completely on mobile devices, so I'm thinking that setting desktop browsers to `false` is the right way to go about this?

Reply:
Yeah, from https://en.wikipedia.org/wiki/PVRTC it's clear that this is only for devices with PowerVR GPUs, and that seems to be a mobile/tablet thing only. Let's set desktop to false.

Reply:
```suggestion
            "version_added": false
```